docs/en/development-and-contribution/build-and-use-agent.md

```diff
@@ -0,0 +1,43 @@
+# Build and use the Agent from source codes
+
+When you want to build and use the Agent from source code, please follow these steps.
+
+## Install SkyWalking Go
+
+Use `go get` to import the latest version of `skywalking-go` program.
+
+```shell
+go get github.com/apache/skywalking-go@latest
```

Comment:
```suggestion
// latest or any commit ID
go get github.com/apache/skywalking-go@latest
```

Reply:
updated, I think we need add a comment when build the agent for make sure they using the same version(commit ID).